src/status_im2/contexts/wallet/scan_account/style.cljs

```diff
@@ -0,0 +1,152 @@
+(ns status-im2.contexts.wallet.scan-account.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def background
+  {:position         :absolute
+   :top              0
+   :bottom           0
+   :left             0
+   :right            0
+   :background-color colors/neutral-95})
+
+(def screen-padding 20)
+(def flash-button-size 32)
```

Comment:
Seems like we're copy pasting a lot here. Would it not be better to abstract this out to a common component ? we can use on the syncing screens too then. We also use a similar designs on the "scan" screens in the shell design files

Reply:
Yes, agree. I'll work on that, and request manual QA so we make sure we are not breaking functionality

Reply:
BTW, I am not sure this would work for syncing screen in the onboarding as that screen has lots of animations code specific for that screen, but will be useful for other parts of the application where we need to scan a QR code

Reply:
yep, even if we can abstract some parts out into common components, e.g that cornered border on the qr scanner is quite custom but used on all of these screens.

Reply:
I've abstracted the entire screen by now, but better to abstract also components like that one you mentioned . I didn't want to make this PR larger than expected so created a separate issue to follow up on this: #17578